ludwig/schema/model_types/utils.py

```diff
@@ -304,8 +304,8 @@ def set_llm_tokenizers(config: "ModelConfig") -> None:
     if config.model_type != "llm":
         return
 
-    pretrained_model_name_or_path = config.model_name
-    if pretrained_model_name_or_path is None:
+    pretrained_model_name_or_path = config.base_model
+    if not isinstance(pretrained_model_name_or_path, str) or pretrained_model_name_or_path is None:
         raise ValueError("Must set model_name when using the LLM model.")
```

Comment:
Update this to be "Must set `base_model` when using the LLM model type"?

Reply:
fixed